### ISSUE TITLE
Mintrayr doesn't work with Firefox "Quantum" v57.0

### ISSUE THREAD

**Issue by blisterbutt** [nmaier/mintrayr#185]
As above.

**Comment by qwertychouskie** [comment]
+1

**Comment by ghost** [comment]
Please, implement Quantum!!!

**Comment by TheSquirrelMafia** [comment]
Yes please! This is my favorite add-on 'cause it comes in handy at work. (^_^ )

**Comment by rexindan** [comment]
+1 to all messages.

**Comment by batagy** [comment]
Yes, this is a must-have addon, that is really missing from Firefox 57+.  It would be a great help if you could update it as a Webextension.

**Comment by lexterror** [comment]
It would be great if it worked with Firefox "Quantum" v57.0...

**Comment by dubecho** [comment]
Please please please update this for new Firefox, must have extension and missed : (

**Comment by njushal** [comment]
+1

**Comment by taword** [comment]
+1

**Comment by smmr0** [comment]
Please use the 👍  reaction on the original comment instead of adding a new comment. That way people can subscribe to this issue and be notified only about relevant information (e.g. responses from the developer, suggestions for alternatives, etc.)

**Comment by qwertychouskie** [comment]
On the contrary, if the dev's inbox gets spammed with enough +1s, maybe @nmaier will update the addon just to restore sanity to their inbox ;)

**Comment by smmr0** [comment]
@qwertychouskie Lol true, but not without spamming my inbox (and 10 other participants' inboxes) at the same time

**Comment by jcvernaleo** [comment]
So I was looking into this a little (since I really miss mintrayr) and I have some doubts as to if it is possible to update the extension.  Specifically, lookng at https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Porting_a_legacy_Firefox_add-on and https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Comparison_with_XUL_XPCOM_extensions since webextensions give less (by design) control over the UI than XUL extensions did.

I'm not saying it is impossible (I haven't spent too much time on this), but there might be some serious rethinking needed.

Has anyone (@nmaier maybe) looked at the new API in any detail and have any ideas?  I'm willing to help if there is a clear path forward, but I'm not sure what (if anything) that path is at the moment.
